leaderelection/leader_election.go

```diff
@@ -39,6 +40,9 @@ const (
 	defaultLeaseDuration = 15 * time.Second
 	defaultRenewDeadline = 10 * time.Second
 	defaultRetryPeriod   = 5 * time.Second
+	healthCheckTimeout   = 20 * time.Second
```

Comment:
should we make this configurable?

Reply:
Leaning towards less knobs until there's a need but no strong preference either way

Reply:
There's at least one ask to make these lease values configurable: https://github.com/kubernetes-csi/csi-lib-utils/issues/62

Reply:
Done